### PR TITLE
FLAP-144 transitions between nodes

### PR DESCRIPTION
**Pull request by amneet270**
# Description

FLAP-144 Machinist-UI: Test to check the ability to draw connections between nodes

test a single transition between 2 nodes and check if the transitions is added to the node_menu feature 

## Link to the work item

relates to FLAP-53 [Machinist-UI] Ability to draw connections between nodes

### CODE REVIEW

**Comment by theNafsters** [comment]
Good job amneet, everything looks great. For the time being, we will leave the main page constant on the file. However, i made a constant under the options.py for the home page (since it is a constant used for every single python file). So, once that is merged, you will have to remove "MAIN_PAGE" from your files and refactor it.

**Review by theNafsters — APPROVED** [review]

**Comment by amneet270** [comment]
> Good job amneet, everything looks great. For the time being, we will leave the main page constant on the file. However, i made a constant under the options.py for the home page (since it is a constant used for every single python file). So, once that is merged, you will have to remove "MAIN_PAGE" from your files and refactor it.

ok @theNafsters I will change it

File: tests/test_transitions.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
+from selenium.webdriver.common.by import By
+from selenium.webdriver.remote.webdriver import WebDriver
+from selenium.webdriver.common.action_chains import ActionChains
+
+MAIN_PAGE = "https://machinist.flapflap.io/"
+
+
+def test_transitioning_nodes(driver: WebDriver):
```
Comment (obonobo) [inline tests/test_transitions.py:35]: Hey @amneet270, what do you think about testing two way transition? Like here we are testing a single transition on the page, an improvement to the test would be to test if a transition could also be made in the reverse direction.

Also, testing creating transitions when there are more nodes on the canvas too. Like 3 nodes.

Comment (amneet270) [reply tests/test_transitions.py:35]: I was going to do that as a different task but you are right it make more sense to directly do it here I will make the required changes

**Pull request merged**